ofctrl/fgraphGroup.go

```diff
@@ -109,6 +118,16 @@ func (self *Group) getGroupModMessage(command int) *openflow15.GroupMod {
 		groupMod.Type = openflow15.GT_FF
 	}
 
+	if command == openflow15.OFPGC_DELETE {
+		return groupMod
+	}
+
+	if command == openflow15.OFPGC_ADD || command == openflow15.OFPGC_MODIFY {
```

Comment:
Is there something wrong to set groupMod.Properties when command is openflow15.OFPGC_DELETE?

Reply:
Yes, OVS will return error message if groupMod.Properties is not nil in deletion command